samples/javascript_nodejs/21.corebot-app-insights/dialogs/bookingDialog.js

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { TimexProperty } = require('@microsoft/recognizers-text-data-types-timex-expression');
+const { InputHints, MessageFactory } = require('botbuilder');
+const { ConfirmPrompt, TextPrompt, WaterfallDialog } = require('botbuilder-dialogs');
+const { CancelAndHelpDialog } = require('./cancelAndHelpDialog');
+const { DateResolverDialog } = require('./dateResolverDialog');
+
+const CONFIRM_PROMPT = 'confirmPrompt';
+const DATE_RESOLVER_DIALOG = 'dateResolverDialog';
+const TEXT_PROMPT = 'textPrompt';
+const WATERFALL_DIALOG = 'waterfallDialog';
+
+class BookingDialog extends CancelAndHelpDialog {
```

Comment:
Please revert the entire constructor back to the original coreBot code.  The only changes are those changes required to add telemetry.  

The original coreBot  BookingDialog constructor:

```JavaScript
    constructor(id) {
        super(id || 'bookingDialog');

        this.addDialog(new TextPrompt(TEXT_PROMPT))
            .addDialog(new ConfirmPrompt(CONFIRM_PROMPT))
            .addDialog(new DateResolverDialog(DATE_RESOLVER_DIALOG))
            .addDialog(new WaterfallDialog(WATERFALL_DIALOG, [
                this.destinationStep.bind(this),
                this.originStep.bind(this),
                this.travelDateStep.bind(this),
                this.confirmStep.bind(this),
                this.finalStep.bind(this)
            ]));

        this.initialDialogId = WATERFALL_DIALOG;
    }
```

Reply:
Hi @WashingtonKayaker, we just applied the feedback.